### PR TITLE
Update CSV.write writeheader parameter to header

### LINKED ISSUE #503: Update deprecated `writeheader=true` at `Domain.jl` `model_spec` function
As in title. 

When running the tests I got the following warning about this:

```
Warning: `writeheader=true` is deprecated in favor of `header=true`
│   caller = #101 at write.jl:161 [inlined]
└ @ Core C:\Users\pribeiro\.julia\packages\CSV\OnldF\src\write.jl:161
```

### PR DESCRIPTION
**Pull request by Zapiano**
As in title.

Closes #503

### CODE REVIEW

File: src/Domain.jl [hunk 1]
```diff
@@ -107,7 +107,7 @@ function model_spec(d::Domain, filepath::String)::Nothing
         write(io, "# Generated with ADRIA.jl $(vers_id) on $(replace(string(now()), "T"=>"_", ":"=>"_", "."=>"_"))\n")
     end
 
-    model_spec(d) |> CSV.write(filepath, writeheader=true, append=true)
+    CSV.write(filepath; header=true, append=true)(model_spec(d))
```
Comment (ConnectedSystems) [inline src/Domain.jl:110]: If we're getting rid of the piping anyway...

```suggestion
    CSV.write(filepath, model_spec(d); header=true, append=true)
```

File: src/Domain.jl [hunk 2]
```diff
@@ -107,7 +107,7 @@ function model_spec(d::Domain, filepath::String)::Nothing
...
 
     return
```
Comment (ConnectedSystems) [inline src/Domain.jl:112]:
```suggestion
    return nothing
```

**Review by ConnectedSystems — CHANGES_REQUESTED** [review]
Some minor additional cleanup and adjustment to Blue style

**Comment by Zapiano** [comment]
@ConnectedSystems  ready for another review.

**Review by ConnectedSystems — APPROVED** [review]
Thank you!

**Pull request merged**